include/vapor/Grid.h

```diff
@@ -1209,6 +1210,16 @@ class VDF_API Grid {
         }
     }
 
+    virtual void ClampIndex(const std::array<size_t, 3> &dims, size_t nNodeDims, const Size_tArr3 indices, Size_tArr3 &cIndices) const
+    {
+        cIndices = {0, 0, 0};
+
+        for (int i = 0; i < nNodeDims; i++) {
```

Comment:
well, I don't understand the logic of the task performed here, so I wrote it in a way that's equivalent to the old code. If you can tell me that it will do the right thing even when the length is fixed to 3, then I can definitely remove the length parameter.

Reply:
It will do the right thing, but you might want to  look at the code and the documentation to convince yourself.

Reply:
Got it, fixed in 52081bac1